UnityProject/Assets/Scripts/Core/Chat/ChatUI.cs

```diff
@@ -166,7 +172,6 @@ public override void Awake()
 		private static readonly List<ChatChannel> RadioChannels = new List<ChatChannel>
 		{
 			ChatChannel.Common,
-			ChatChannel.Binary,
```

Comment:
wouldn't silicons want to talk to each other on the radio sometime?

Reply:
I moved it out of there as toggling it toggled on local too:

![image](https://user-images.githubusercontent.com/56727168/181279193-f3400af1-129b-47f3-816d-a31d27228094.png)